### DISCUSSION TITLE
Documentation en français

### DISCUSSION THREAD

**Discussion by Peio47** [robot-soccer-kit/robot-soccer-kit#45] [Show and tell]
Bonjour,
J'ai réalisé une [documentation ](https://gitlab.com/Peio47/robot-soccer-kit/-/blob/main/documentation.pdf) qui reprend la documentation en ligne en anglais et qui ajoute quelques diagrammes et explications.

Bonne lecture

**Comment by RVDROU** [comment]
Merci pour cette ressource ...

**Comment by Moumout24** [comment]
Merci Pierre pour la ressources.
